format.py

```diff
@@ -0,0 +1,13 @@
+name = "carlos"
+age = 22
+
+print("hello {} you are {} years old .\n".format(name ,age))
+
+
+#second form
+
+print("hello {name} you are {age} years old .".format(name = "carlos" ,age = 22))
+
+#third form
+
+print("hello {0} you are {1} years old .".format(name ,age))

```

Comment:
commas should be next the first word, for instance:
`(name, age)`

Reply:
i did it